### PR TITLE
Add guide for running the txn processor locally

### PR DESCRIPTION
**Pull request by peterschwarz**
Signed-off-by: Peter Schwarz <pschwarz@bitwise.io>

### CODE REVIEW

File: docs/source/txn_processor_dev.rst [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+.. code-block:: console
+
+  $ docker-compose -f omi-summer-lab/docker/compose-without-omi.yaml up
+
+To stop the validator node, press Ctrl-c and then run:
+
+.. code-block:: console
+
+  $ docker-compose -f omi-summer-lab/docker/compose-with-omi.yaml down
```
Comment (jsmitchell) [inline docs/source/txn_processor_dev.rst]: without

File: docs/source/txn_processor_dev.rst [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+validator we started in the previous section. 
+
+Making changes to the Transaction Processor
+-------------------------------------------
+
+Modifications to the Transaction Processor can be made by editing the
+Transaction Handler implementation found at
+`omi-summer-lab/omi/sawtooth_omi/handler.py`.  The important method to focus on
+in this file is `OMITransactionHandler.apply`. This is where the operations of
+validating the transaction payload and setting  global state values happen.
```
Comment (jsmitchell) [inline docs/source/txn_processor_dev.rst]: s/setting  global/setting global/

**Review by jsmitchell — CHANGES_REQUESTED** [review]

File: docs/source/txn_processor_dev.rst [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+with the remaining items needed. In the `omi-summer-lab` directory, run:
+
+.. code-block:: console
+
+  $ pip3 install -r requirements.txt
+
+Running the Transaction Processor
+---------------------------------
+
+We can now run the Transaction Processor. In the `omi-summer-lab` direcotry,
```
Comment (nick-drozd) [inline docs/source/txn_processor_dev.rst]: directory

File: docs/source/txn_processor_dev.rst [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+
+  See the Application Developer's Guide section on `The apply Method
+  <https://intelledger.github.io/_autogen/sdk_TP_tutorial_python.html#the-apply-method>`__
+  for more information.
+
+Currently, this handler implementation writes OMI objects to the global state,
+and validates ownership based on the public key associated with the enclosing
+transaction.
+
+You could, for example, build sample payment record transaction for a number of
```
Comment (nick-drozd) [inline docs/source/txn_processor_dev.rst]: build a sample

File: docs/source/txn_processor_dev.rst [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+plays of a song from a streaming service. The transaction would contain the
+song identifier and the number of plays and the total payment and would be
+signed by the streaming service identity (i.e. “youtube” or “spotify”). The
```
Comment (nick-drozd) [inline docs/source/txn_processor_dev.rst]: e.g.

**Review by nick-drozd — CHANGES_REQUESTED** [review]

**Review by jsmitchell — APPROVED** [review]

**Review by nick-drozd — APPROVED** [review]

**Pull request merged**